report/web.py

```diff
@@ -85,8 +88,8 @@ def get_results(benchmark) -> tuple[list[evaluator.Result], list[str]]:
   return results, targets
 
 
-def get_prompt(benchmark) -> Optional[str]:
-  root_dir = os.path.join(RESULTS_DIR, benchmark)
+def get_prompt(bnmk) -> Optional[str]:
```

Comment:
Why is this rename required?

Reply:
To avoid linting warnings:
```
Redefining name 'benchmark' from outer scope
```
This is becase we have a function named `benchmark`:
https://github.com/google/oss-fuzz-gen/blob/main/report/web.py#L232

Reply:
Can we instead rename that function? `bnmk` is a bit inconsistent with all of our other naming.

Reply:
OK!

Reply:
I will do the same with `sample`

Reply:
Fixed